packages/clayui.com/src/pages/docs/advanced_guides/using-js-components-with-metal.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Using JS Components with Metal.js"
+description: ""
+layout: "guide"
+weight: 1
+---
+
+<article id="using-js-components-with-metal">
+
+Clay Components is made according to Lexicon standards and built on top of the [Metal.js](https://metaljs.com/) framework, so you can take advantage of Clay's components in your two template systems.
+
+## Jsx
+
+You can import your components as described in [Importing the JS Component](/docs/getting_started/importing-the-js-component.html), consider:
+
+
+```javascript
+import {ClayBadge} from 'clay';
+```
+
+In JSX components call only one component of Clay as any JSX component in Metal.js, consider:
+
+```javascript
+import Component from 'metal-jsx';
+import {ClayBadge} from 'clay';
+
+class App extends Component {
+    render() {
+        return(
+            <div>
+                <h4>My Application with Clay<h4>
+                <hr />
+                <ClayBadge
```

Comment:
Hey @matuzalemsteles, I think it's worth mentioning here that this is a `WebComponent` and not a React component, right?

Reply:
Hey @jbalsas, In that case it's actually a Soy component but it can be used inside a JSX component in Metal.js. But maybe I should make it clearer that this happens only on Metal.js.

Reply:
🤔 I didn't remember we could do that 😂

Reply:
I did some testing with this to see if the Analytics team could use some of Clay's components, we just can not use a JSX component on Soy (I have some ideas for this to be possible).